core/src/test/java/io/parsingdata/metal/expression/value/reference/CurrentIterationTest.java

```diff
@@ -53,6 +56,7 @@
             { "[0, 1, 2, 3] repn=4(CURRENT_ITERATION)", repn(VALUE_EQ_ITERATION, con(4)), stream(0, 1, 2, 3), enc(), true },
             { "[255, 0, 1, 2, 3, 255] def(255), while<3(CURRENT_ITERATION), def (255)", seq(VALUE_EQ_255, whl(VALUE_EQ_ITERATION, not(eq(con(3)))), VALUE_EQ_255), stream(255, 0, 1, 2, 3, 255), enc(), true },
             { "[0, 0, 1, 2, 255, 1, 0, 1, 2, 255] repn=2(CURRENT_ITERATION, repn=3(CURRENT_ITERATION))", repn(seq(VALUE_EQ_ITERATION, repn(VALUE_EQ_ITERATION, con(3)), VALUE_EQ_255), con(2)), stream(0, 0, 1, 2, 255, 1, 0, 1, 2, 255), enc(), true },
+            { "[0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2] repn=2(CURRENT_ITERATION, repn=2(PARENT_ITERATION, repn=2(GRANDPARENT_ITERATION)))", repn(seq(VALUE_EQ_ITERATION, repn(seq(VALUE_EQ_ITERATION_PARENT, repn(VALUE_EQ_ITERATION_GRANDPARENT, con(2))), con(1))), con(3)), stream(0, 0, 0, 0, 1, 1, 1, 1, 2, 2, 2, 2), enc(), true },
```

Comment:
The description of the test does not match the actual `Token`, right? Description shows `repn2(repn2(repn2 ))` but the actual `Token` shows `repn3(repn1(repn2))`

Reply:
Correct. Adjusted it.